include/mbedtls/ssl.h

```diff
@@ -493,10 +493,11 @@ typedef int mbedtls_ssl_send_t( void *ctx,
  * \param buf      Buffer to write the received data to
  * \param len      Length of the receive buffer
  *
- * \return         The callback must return the number of bytes received,
- *                 or a non-zero error code.
- *                 If performing non-blocking I/O, \c MBEDTLS_ERR_SSL_WANT_READ
+ * \returns        If data has been received, the positive number of bytes received.
+ * \returns        \c 0 if the connection has been closed.
+ * \returns        If performing non-blocking I/O, \c MBEDTLS_ERR_SSL_WANT_READ
```

Comment:
Do we need the `\returns` on line 498? I feel like we don't.

Reply:
We often use the convention of one `\return` directive per error code. Why not?

Reply:
It's taste to some extent - I personally find multiple `\returns` statements more readable.

Reply:
OK, then.